src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+    </div>
+
+    <div class="stars stars--1">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+    </div>
+
+    <div class="stars stars--2">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+    </div>
+
+    <div class="stars stars--3">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+    </div>
+
+    <div class="stars stars--4">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+    </div>
+
+    <div class="stars stars--5">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+    </div>
+
   </body>
+
 </html>
```

Comment:
```suggestion
    </div>
  </body>
</html>
```
Don't leave empty lines between parent and child elements